src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -0,0 +1,1773 @@
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Inline Exercise:</b>\n",
+    "We have now defined all the feed conditions and the inputs required for the unit models. The system should now have 0 degrees of freedom i.e. should be a square problem. Please check that the degrees of freedom is 0. \n",
+    "\n",
+    "Use Shift+Enter to run the cell once you have typed in your code. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "exercise"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Check the degrees of freedom\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Check the degrees of freedom\n",
+    "print(degrees_of_freedom(m))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Check the degrees of freedom\n",
+    "assert degrees_of_freedom(m) == 0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initialization\n",
+    "\n",
+    "This subsection will demonstrate how to use the built-in sequential decomposition tool to initialize our flowsheet.\n",
+    "\n",
+    "Let us first create an object for the `SequentialDecomposition` and specify our options for this. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seq = SequentialDecomposition()\n",
+    "seq.options.select_tear_method = \"heuristic\"\n",
+    "seq.options.tear_method = \"Wegstein\"\n",
+    "seq.options.iterLim = 5\n",
+    "\n",
+    "# Using the SD tool\n",
+    "G = seq.create_graph(m)\n",
+    "heuristic_tear_set = seq.tear_set_arcs(G, method=\"heuristic\")\n",
+    "order = seq.calculation_order(G)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Which is the tear stream? Display tear set and order"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for o in heuristic_tear_set:\n",
+    "    print(o.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "What sequence did the SD tool determine to solve this flowsheet with the least number of tears? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for o in order:\n",
+    "    print(o[0].name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The SequentialDecomposition tool has determined that the tear stream is the mixer outlet (s03 in the Figure above). We will need to provide a reasonable guess for this."
```

Comment:
I think we should add an explanation of how we got the guess too - it is the sum of the two feed streams (i.e. assuming negligible recycle for the first pass).

Reply:
Added the following explanation:
"For the initial guess, we assume that the flowrate of the recycle stream (s09) is zero. Consequently, the flow rate of the stream s03 is simply the sum of the flowrates of the toluene feed and hydrogen feed streams. Further, since the temperature and the pressure of both the toluene and hydrogen feed streams are the same, we specify their values as the initial guess for the temperature and pressure of the stream s03."